File: app/src/main/java/com/airfreshener/telegram_sms/data_structure/polling_json.java
package com.airfreshener.telegram_sms.data_structure;

public class polling_json {
    //Predefined types that accept return
    @SuppressWarnings({"unused", "RedundantSuppression"})
    public final String[] allowed_updates = {"message", "channel_post", "callback_query"};
    public long offset;
    public int timeout;
}

File: app/src/main/java/com/airfreshener/telegram_sms/model/PollingJson.kt
package com.airfreshener.telegram_sms.model

class PollingJson {
    //Predefined types that accept return
    @Suppress("unused", "PropertyName")
    val allowed_updates = arrayOf("message", "channel_post", "callback_query")
    @JvmField
    var offset: Long = 0
    @JvmField
    var timeout = 0
}
